Packages/ChatSecure/ChatSecure/Classes/Networking/ChannelStorage.swift

```diff
@@ -7,18 +7,25 @@
 
 import Foundation
 
-class ChannelStorage {
-	static let shared = ChannelStorage()
-	
-	private var channels: [String: APIService]
+public protocol IChannelStorage {
+	var channels: [String: APIService] { get }
+	var config: IChatSecureConfig { get }
+}
+
+public class ChannelStorage: IChannelStorage {
+	public var channels: [String: APIService]
+	public let config: IChatSecureConfig
 	
-	private init() {
-		channels = [Constants.clkDomain: APIService(domain: Constants.clkDomain)]
+	public init(config: IChatSecureConfig) {
+		self.config = config
+		channels = [config.clkDomain + config.clkPort: APIService(domain: config.clkDomain + config.clkPort)]
 	}
 	
-	func getChannels(domain: String) -> APIService {
+	public func getChannels(domain: String) -> APIService {
 		if channels.contains(where: { $0.key == domain }) {
 			return channels[domain] ?? APIService(domain: domain)
+		} else if domain.isEmpty {
+			return channels[config.clkDomain + config.clkPort]  ?? APIService(domain: config.clkDomain + config.clkPort)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/03ed79698ba6e18faa1dddb6e6524285a668b75e/Packages/ChatSecure/ChatSecure/Classes/Networking/ChannelStorage.swift#L28"></span>Operators should be surrounded by a single whitespace when they are being used.<br />
<code>operator_usage_whitespace</code> <code>ChannelStorage.swift:28</code></td>
    </tr>
  </tbody>
</table>